plugins/typo/src/main.rs

```diff
@@ -66,34 +67,25 @@ impl TryFrom<RawConfig> for Config {
 async fn typo(engine: &mut PluginEngine, value: Target) -> Result<Vec<bool>> {
 	log::debug!("running typo query");
 
-	let local = value.local;
-
-	// Get the typo file
-	let typo_file = TYPOFILE.get().ok_or(Error::UnspecifiedQueryState)?;
+	// Get the typo file.
+	let typo_file = TYPOFILE
+		.get()
+		.ok_or_else(|| anyhow!("could not find typo file"))?;
 
 	// Get the repo's dependencies
 	let value = engine
-		.query("mitre/npm/dependencies", local)
+		.query("mitre/npm/dependencies", value.local)
 		.await
-		.map_err(|e| {
-			log::error!("failed to get dependencies for typo query: {}", e);
-			Error::UnspecifiedQueryState
-		})?;
+		.map_err(Error::any)?;
```

Comment:
In the `review` plugin changes, you used `.context()` to explain "failed to get pull request reviews". Why not do the analogous thing here?

Reply:
Oh fair point. I can make that edit